codegen/types/templates/api.tmpl

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python3
```

Comment:
[pebble] It feels weird to me to have a shebang line on a non-executable file. Can we remove this from the template so it doesn't generate into the library files?

Reply:
Yeah, fair point, just habit to put it on every new python file